src/python/pants/backend/python/rules/run_setup_py_test.py

```diff
@@ -104,7 +104,7 @@ def assert_error(self, addr: str, exc_cls: Type[Exception]):
                 SetupPyChroot,
                 Params(
                     SetupPyChrootRequest(ExportedTarget(self.tgt(addr)), py2=False),
-                    SourceRootConfig.global_instance(),
+                    create_options_bootstrapper(args=["--source-root-patterns=src/python"]),
```

Comment:
Is this pattern no longer built in?

Reply:
No. All patterns must be explicitly specified now.